parsing.indexing/test/unit/src/org/netbeans/modules/parsing/impl/indexing/ScanStartedTest.java

```diff
@@ -84,12 +88,12 @@ protected void getAdditionalServices(List<Class> clazz) {
         super.getAdditionalServices(clazz);
         clazz.add(FooCPP.class);
     }
-
-    final TestHandler handler = new TestHandler();
```

Comment:
The handler could well remain as a final variable, I think. Each testcase creates a new instance of this class.